services/supervisor/src/lib.rs

```diff
@@ -531,9 +531,11 @@ impl Supervisor {
                 if let Some(tx_sender) = tx_sender {
```

Comment:
IMO, we should consider making `start_deploy` accept futures; blocking in the handler looks weird.

Reply:
I thought about it, but anyway in current testkit implementation we should wait until this future will be finished.